lib/maintainers.nix

```diff
@@ -20,6 +20,12 @@
     name = "Daniel Laing";
     keys = [ { fingerprint = "0821 8B96 DC73 85E5 BB7C  A535 D264 3BD2 13BC 0FA8"; } ];
   };
+  elythh = {
+    email = "gwen@omg.lol";
+    github = "elythh";
+    githubId = 50964650;
+    name = "gwen";
+  };
```

Comment:
FYI @khaneliman: ideally we like new maintainers to be in their own commit, separate from the new plugin.

Reply:
Ugh don’t know how I missed that

Reply:
Easily done. Commits + commit messages are hidden out of the way when reviewing on GitHub, most of the time we just focus on the overall diff.